### DISCUSSION TITLE
CC2652R1 firmware?

### DISCUSSION THREAD

**Discussion by agrieco** [Koenkk/zigbee2mqtt#7974] [Devices]
Can someone confirm which coordinator firmware I should use for this device?  https://github.com/Koenkk/Z-Stack-firmware/blob/master/coordinator/Z-Stack_3.x.0/bin/CC2652R_coordinator_20210120.zip is what I think is the right one?  

I'm having some massive issues with this one and the dreaded Failed to connect to the adapter (Error: SRSP - SYS - ping after 6000ms).  I've done lots of debugging so I'm going back to the basics...

thanks

**Comment by Theminijohn** [comment]
That's the right one 👍  I'm having the same issues with mine. In my instance, I narrowed it down to the reset button needing to be pressed when first plugged in and on every restart.

I'm still battling with this tho. It was working for a couple of months and after a power outage, I couldn't get it connected anymore.

Haven't found much information about this either :/
